roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -11,6 +11,13 @@
   set_fact:
     is_ostree: "{{ ostree.stat.exists }}"
 
+- name: get ostree version
+  shell: "set -o pipefail && rpm-ostree --version | awk -F\\' '/Version/{print $2}'"
```

Comment:
Why do you still need the `set -o pipefail` ?

Reply:
Ansible-lint [306]: Shells that use pipes should set the pipefail option.

Reply:
I was not aware of this, thanks for the explanation!